app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class GraphqlController < ApplicationController
+  # If accessing from outside this domain, nullify the session
+  # This allows for outside API access while preventing CSRF attacks,
+  # but you'll have to authenticate your user separately
+  # protect_from_forgery with: :null_session
```

Comment:
this is not being applied should we uncomment it? should we use 
```suggestion
  protect_from_forgery with: :exception
```

Reply:
Ok, let me check it out. Maybe by now won't be necessary or possible. Actually we are using `skip_before_action :verify_authenticity_token`.

Reply:
@tramuntanal Hi! As I was suggesting, right now we don't have an authentication method developed. It was not part of the task, so we need to keep it as it is by now.

Thank you!